static/js/notifications.js

```diff
@@ -30,7 +30,9 @@ if (window.webkitNotifications) {
                 return 2;
             }
         },
-        requestPermission: window.Notification.requestPermission,
+        requestPermission: function (callback) {
+            window.Notification.requestPermission(callback);
+        },
```

Comment:
I'm still not sure I understand why this change is needed at all -- doesn't the pre-patch code work do the exact same thing (aside from the detail that the post-patch version doesn't pass on whatever requestPermission returns).